### PR TITLE
Adds new option "useClassicEngine"

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## Description:

Sets OIE on "by default". OIDC flow will use interaction code flow by default. New option "useClassicEngine" can be set to true to enable embedded authn flows.

- removes option "useInteractionCodeFlow"
- adds new option "useClassicEngine" (false by default)

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [x] Added unit tests?

- [x] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

- Did you verify the change by running downstream monolith artifacts? (YES | NO | UNSURE)

### Issue:

- [OKTA-487653](https://oktainc.atlassian.net/browse/OKTA-487653)

### CODE REVIEW

File: src/v2/view-builder/utils/AuthenticatorUtil.js [hunk 1]
```diff
@@ -184,6 +184,7 @@ const getAuthenticatorData = function(authenticator, isVerifyAuthenticator) {
       description: isVerifyAuthenticator
         ? authenticator?.relatesTo?.displayName
         : loc('oie.custom.app.authenticator.description', 'login', [authenticator.label]),
+      noTranslateClassName: isVerifyAuthenticator ? 'no-translate' : '',
```
Comment (jaredperreault-okta) [inline src/v2/view-builder/utils/AuthenticatorUtil.js:187]: Seems unrelated to PR - is this on purpose?

**Review by jaredperreault-okta — APPROVED** [review]

File: src/v2/view-builder/utils/AuthenticatorUtil.js (hunk 1 above)
Comment (aarongranick-okta) [reply src/v2/view-builder/utils/AuthenticatorUtil.js:187]: Yes, this is a separate issue I found. Must be hidden by false positives on english leaks. I will cherry-pick this over to a separate PR

Comment (aarongranick-okta) [reply src/v2/view-builder/utils/AuthenticatorUtil.js:187]: https://github.com/okta/okta-signin-widget/pull/2564
